content/security/for-admins/scim.md

```diff
@@ -58,3 +61,196 @@ You must make sure you have [configured SSO](single-sign-on/configure/_index.md)
+
+After you set the role in the IdP, you need to sync to push the changes to Docker.
+
+The external namespace to use to set up these attributes is `urn:ietf:params:scim:schemas:extension:docker:2.0:User`.
+
+{{< tabs >}}
+{{< tab name="Okta" >}}
+
+### Set up
+
+1. Setup [SSO](./single-sign-on/configure/_index.md) and SCIM first.
+2. In the Okta admin portal, go to **Directory > Profile Editor** and click on **User(Default)**.
+3. Select **Add Attribute** and configure the values for the role, org, or team you want to add. Exact naming isn't required.
+4. Return to the **Profile Editor** and select your application.
+5. Select **Add Attribute** and enter the required values. The **External Name** and **External Namespace** must be correct. The external name values for org/team/role mapping are `dockerOrg`, `dockerTeam`, and `dockerRole` respectively, as listed in the previous table. The external namespace is the same for all of them: `urn:ietf:params:scim:schemas:extension:docker:2.0:User`.
+6. After creating the attributes, go to the top and select **Mappings > Okta User to YOUR APP**.
+7. Go to the newly created attributes and map the variable names selected above to the external names, then select **Save Mappings**. If you’re using JIT provisioning, continue to the following step.
+8. Go to **Applications > YOUR APP > General > SAML Settings > Edit > Step 2** and configure the mapping from the user attribute to the docker variables.
+
+### Assign roles by user
+
+1. Go to **Directory > People > YOUR USER > Profile**, then select **Edit** on **Attributes**.
+2. Update the attributes to the desired values.
+
+### Assign roles by group
+
+1. Go to **Directory > People > YOUR GROUP > Applications > YOUR APPLICATION**, then select the **Edit** icon.
+2. Update the attributes to the desired values.
+
+If a user doesn't already have attributes set up, users who are added to the group will inherit these attributes upon provsioning.
+
+{{< /tab >}}
+{{< tab name="Entra ID SAML 2.0" >}}
+
+### Set up
+
+1. Setup [SSO](./single-sign-on/configure/_index.md) and SCIM first.
+2. In the Azure AD admin portal, go to**Enterprise Apps > YOUR APP > Provisioning > Mappings > Provision Azure Active Directory Users**.
+3. To setup up the new mapping, check **Show advanced options** and then select **Edit attribute options**.
+4. Create new entries with the desired mapping for role, org, or group (for example, `urn:ietf:params:scim:schemas:extension:docker:2.0:User:dockerRole`) as a string type.
+5. Go back to **Attribute Mapping** for users and click **Add new mapping**.
+
+### Expression mapping
```

Comment:
**Note to technical reviewer:** Let me know if there are any missing steps here! In our internal guide most of this info is obtained from the configuration screenshot.

Reply:
wondering if we should omit OneLogin docs as it is quite complex, we don't have many customers that use it and we don't have role and group mapping specific docs for this idp .. 🤔

Reply:
That's a good question, I'll check with PMs to see if omitting OneLogin docs would be the best approach here. If anything, we can release without OneLogin to start and if we get a lot of feedback requesting it, include it later.

Reply:
I checked with @ajthilakan and @francesco-corti - they are fine removing OneLogin from these docs. I'll make that change today, which I think covers your feedback here. Thanks @ivan-californias  👍